pxe_manager/pxemanager.py

```diff
@@ -65,8 +65,9 @@ def __init__(self, cobbler_url, cobbler_user, cobbler_password, host_manager_cli
         self.private_ip_manager = private_ip_manager_client
         self.ssh_user = ssh_user
         self.ssh_password = ssh_password
-        self.distro = {'centos': 'centos7-x86_64-raid0',
+        self.distro = {'centos': 'centos7u3-x86_64-raid0',
```

Comment:
Can we please make centos = centos7 to give us the latest centos as we do for RHEL. then add a centos7u3=centos7u3-....

I think it's way cleaner and follows with our pattern.